Library/Formula/zurl.rb

```diff
@@ -0,0 +1,21 @@
+class Zurl < Formula
+  homepage "https://github.com/fanout/zurl"
+  url "http://packages.fanout.io/source/zurl-1.3.1.tar.bz2"
+  sha1 "8a04ad092a4fd9b174a352b910c190466fc39eb9"
+
+  depends_on "pkg-config" => :build
+  depends_on "curl"
```

Comment:
Might be worth doing ` depends_on "curl" if MacOS.version < :lion `. I think everything newer than 10.7 is on curl 7.21+ but prior to that probably needs the dep.